ui/src/data_explorer/containers/Header.js

```diff
@@ -2,23 +2,24 @@ import React, {PropTypes} from 'react'
 import {withRouter} from 'react-router'
 
 import AutoRefreshDropdown from 'shared/components/AutoRefreshDropdown'
-import TimeRangeDropdown from '../../shared/components/TimeRangeDropdown'
-import SourceIndicator from '../../shared/components/SourceIndicator'
-import GraphTips from '../../shared/components/GraphTips'
+import TimeRangeDropdown from 'shared/components/TimeRangeDropdown'
+import SourceIndicator from 'shared/components/SourceIndicator'
+import GraphTips from 'shared/components/GraphTips'
 
 const {func, number, shape, string} = PropTypes
 
 const Header = React.createClass({
   propTypes: {
-    autoRefresh: number.isRequired,
-    timeRange: shape({
-      upper: string,
-      lower: string,
-    }).isRequired,
     actions: shape({
       handleChooseAutoRefresh: func.isRequired,
       setTimeRange: func.isRequired,
     }),
+    autoRefresh: number.isRequired,
+    showWriteFormAction: func.isRequired,
```

Comment:
yeah, i'd just import this as `showWriteForm` (or possibly `onShowWriteForm`) since there's no namespace collision